<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_HvONOff</class>
 <widget class="QDialog" name="Dial_HvONOff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_HVOn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch On</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_HvOff">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch Off</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HvOn_0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Hv On</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HvOn_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Hv On</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HvOn_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Hv On</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HvOn_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Hv On</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HVSel_0">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modul 0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HVSel_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modul 1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HVSel_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modul 2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_HVSel_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Modul 3</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Select</string>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>pushButton_HVOn</zorder>
  <zorder>pushButton_HvOff</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>textEdit_Msg</zorder>
  <zorder>checkBox_HvOn_0</zorder>
  <zorder>checkBox_HvOn_2</zorder>
  <zorder>checkBox_HvOn_1</zorder>
  <zorder>checkBox_HvOn_3</zorder>
  <zorder>checkBox_HVSel_0</zorder>
  <zorder>checkBox_HVSel_1</zorder>
  <zorder>checkBox_HVSel_2</zorder>
  <zorder>checkBox_HVSel_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
